app/helpers/application_helper.rb

```diff
@@ -34,6 +34,13 @@ def date_string(start_date, end_date)
     result
   end
 
+  # Returns the time with the conference's timezone
+  def set_offset(time, timezone)
```

Comment:
you don't need to calculate offsets. I think it is quite difficult to understand.

You can just do, getting the time in UTC and adding the conference timezone:

``` Ruby
time.strftime("%F %R") + " " + timezone.to_s
```

This would get something like:

`2017-03-27 09:30 (GMT-10:00) Hawaii`

And if you like that it render the name of the time zone (I personally like it), yo can also use the `parse` method, basically remove the time zone, which is always UTC, and get the time time in this zone. 

``` Ruby
timezone.parse(time.strftime("%F %R"))
```

In this case you'll get something like:

`Mon, 27 Mar 2017 09:30:00 HST -10:00`


@nikhilgupta1211 @differentreality what do you think? I like the first alternative I give much more than calculate offset and the given string is also more readable in my opinion.